src/profile

```diff
@@ -24,30 +24,43 @@ function _glibc_check () {
   # Workaround for if the ChromeOS glibc has been updated from under us
   # by a ChromeOS upgrade.
   # OLD_LD_LIBRARY_PATH=$LD_LIBRARY_PATH
-  crew_installed_glibc_version=$(LD_LIBRARY_PATH= /usr/bin/jq --arg key glibc -r '.installed_packages[] | select(.name == $key )| .version ' $CREW_PREFIX/etc/crew/device.json)
+  crew_installed_glibc_version=$(LD_LIBRARY_PATH= /usr/bin/jq --arg key glibc -r '.installed_packages[] | select(.name == $key )| .version ' "$CREW_PREFIX"/etc/crew/device.json)
   crew_installed_glibc_package_version="${crew_installed_glibc_version//.}"
-  crew_installed_glibc_package_name="glibc_lib${crew_installed_glibc_package_version%-*}"
+  crew_installed_glibc_package_name_a="glibc_lib${crew_installed_glibc_package_version%-*}"
+  crew_installed_glibc_package_name_b="glibc_${crew_installed_glibc_package_version%-*}"
   installed_glibc_version=$(LD_LIBRARY_PATH= /lib"$LIB_SUFFIX"/libc.so.6 | LD_LIBRARY_PATH= /usr/bin/awk -F version 'NR==1{{gsub (" ", "", $0) ; print substr($2, 1, length($2)-1)}}')
   if [ "${crew_installed_glibc_version%-*}" != "$installed_glibc_version" ]; then
-    LD_LIBRARY_PATH= /bin/grep .so $CREW_PREFIX/etc/crew/meta/"${crew_installed_glibc_package_name}".filelist | LD_LIBRARY_PATH=  /bin/grep -v gconv | LD_LIBRARY_PATH= /usr/bin/xargs /bin/rm -rf
+    if [[ -f "$CREW_PREFIX"/etc/crew/meta/"${crew_installed_glibc_package_name_a}".filelist ]]; then
+      crew_installed_glibc_package_filelist="$CREW_PREFIX"/etc/crew/meta/"${crew_installed_glibc_package_name_a}".filelist
+    elif [[ -f "$CREW_PREFIX"/etc/crew/meta/"${crew_installed_glibc_package_name_b}".filelist ]]; then
+      crew_installed_glibc_package_filelist="$CREW_PREFIX"/etc/crew/meta/"${crew_installed_glibc_package_name_b}".filelist
+    else
+      return 0 2>/dev/null || exit 0
```

Comment:
Wouldn't simply `exit 0` work here?  Seems overly complicated.

Reply:
You need a return in case you are sourcing the script.

Reply:
Otherwise it will kill the shell.

Reply:
(have learned this the hard way...)

Reply:
Wouldn't the `|| exit 0` do the same thing?  What is wrong with just letting it return an error and we debug to fix it?  Something like `return $?`?

Reply:
If it ever gets to this point, something is really wrong, right?

Reply:
See how we are doing this in sommelier already: 
https://github.com/chromebrew/chromebrew/blob/6a1f5d15c400b2e3b65a21410628e6ef1fc4a844/packages/sommelier.rb#L339

Reply:
(I should really just copy that code to be uniform...)